docs/docs/graph-recipe.mdx

```diff
@@ -6,7 +6,13 @@ description: Learn about Graph Recipe for Rasa Open Source.
 abstract: Graph recipes provide a more fine tuned configuration for your executable graphs.
 ---
 
-We now support graph recipes in addition to the default recipe. What graph recipes provide is more powerful control over how execution graph schemas are built.
+:::tip Default Recipe or Graph Recipe?
+
+You will probably only need graph recipes if you're running ML experiments or ablation studies on an existing model. We recommend starting with the default recipe and for many applications that will be all that's needed.
```

Comment:
"ablation studies" feels a bit specific? (I had to google it 😁 )

Reply:
I've used the exact wording from @rctatman actually and I had to google it too :D I think it is good because it has the effect of warning against using graph recipes if the user is not too advanced.